app.js

```diff
@@ -0,0 +1,276 @@
+'use strict'
+
+let seattleSales = document.getElementById('seattle');
+// console.log(cookieElem);
+let timeHours = ['6am','7am','8am','9am','10am','11am','12pm','1pm','2pm','3pm','4pm','5pm','6pm','7pm'];
+
+function custHr(min, max) {
+    return Math.floor(Math.random() * (max - min + 1) + min);
+
+}
+
+let seattle = {
+    cityName: 'Seattle',
+    minCust: 23,
+    maxCust: 65,
+    averageCookies: 6.3,
+    allCustHr: [],
+    cookiePerCust: [],
+    total: 0,
+    getCustHr: function() {
+    for(let i=0; i< timeHours.length; i++){ 
+        let randomCustHr = custHr(this.minCust, this.maxCust)
+        this.allCustHr.push(randomCustHr);
+    }
+},
+    getCookPerCust: function() {

```

Comment:
Great function and variable names! This makes it very easy to read and understand your code.